src/App.css

```diff
@@ -0,0 +1,51 @@
+body {
+  width: 50vw;
+  margin: 0 auto;
+  text-align: center;
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
margin and font-family are the only styles you can store in body, create main container for all other styles